rosdep/base.yaml

```
@@ -795,6 +795,11 @@ emacs:
   gentoo: [virtual/emacs]
   openembedded: [emacs@meta-oe]
   ubuntu: [emacs]
+embree:
+  arch: [embree]
+  debian: [libembree-dev]
+  fedora: [embree]
+  ubuntu: [libembree-dev]
```

the package is not available in xenial and bionic, maybe:

```suggestion
  ubuntu:
    '*': [libembree-dev]
    bionic: null
    xenial: null
```

---

Thank you for pointint out this problem, I fixed in https://github.com/ros/rosdistro/pull/28065/commits/29174c447e57ff9e009f0dc77039ba45cc314207